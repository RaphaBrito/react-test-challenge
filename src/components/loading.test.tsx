import { expect, it } from "vitest";

import { Loading } from "./loading";
import { render } from "../test/utils";

it("should render correctly", () => {
  const result = render(<Loading />);
  expect(result).toMatchSnapshot('<Loading />');
});
