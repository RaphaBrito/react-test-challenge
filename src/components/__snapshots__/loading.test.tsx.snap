// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render correctly > <Loading /> 1`] = `
<div>
  <h1>
    Carregando...
  </h1>
</div>
`;
